src/languages/stan.js

```diff
@@ -1,89 +1,227 @@
+    'num_elements', 'ordered_logistic_lpmf', 'ordered_logistic_rng', 'owens_t',
+    'pareto_cdf', 'pareto_lccdf', 'pareto_lcdf', 'pareto_lpdf', 'pareto_rng',
+    'pareto_type_2_cdf', 'pareto_type_2_lccdf', 'pareto_type_2_lcdf',
+    'pareto_type_2_lpdf', 'pareto_type_2_rng', 'pi', 'poisson_cdf', 'poisson_lccdf',
+    'poisson_lcdf', 'poisson_log_lpmf', 'poisson_log_rng', 'poisson_lpmf',
+    'poisson_rng', 'positive_infinity', 'pow', 'print', 'prod', 'qr_Q', 'qr_R',
+    'quad_form', 'quad_form_diag', 'quad_form_sym', 'rank', 'rayleigh_cdf',
+    'rayleigh_lccdf', 'rayleigh_lcdf', 'rayleigh_lpdf', 'rayleigh_rng', 'reject',
+    'rep_array', 'rep_matrix', 'rep_row_vector', 'rep_vector', 'rising_factorial',
+    'round', 'row', 'rows', 'rows_dot_product', 'rows_dot_self',
+    'scaled_inv_chi_square_cdf', 'scaled_inv_chi_square_lccdf',
+    'scaled_inv_chi_square_lcdf', 'scaled_inv_chi_square_lpdf',
+    'scaled_inv_chi_square_rng', 'sd', 'segment', 'sin', 'singular_values', 'sinh',
+    'size', 'skew_normal_cdf', 'skew_normal_lccdf', 'skew_normal_lcdf',
+    'skew_normal_lpdf', 'skew_normal_rng', 'softmax', 'sort_asc', 'sort_desc',
+    'sort_indices_asc', 'sort_indices_desc', 'sqrt', 'sqrt2', 'square',
+    'squared_distance', 'step', 'student_t_cdf', 'student_t_lccdf',
+    'student_t_lcdf', 'student_t_lpdf', 'student_t_rng', 'sub_col', 'sub_row',
+    'sum', 'tail', 'tan', 'tanh', 'target', 'tcrossprod', 'tgamma', 'to_array_1d',
+    'to_array_2d', 'to_matrix', 'to_row_vector', 'to_vector', 'trace',
+    'trace_gen_quad_form', 'trace_quad_form', 'trigamma', 'trunc', 'uniform_cdf',
+    'uniform_lccdf', 'uniform_lcdf', 'uniform_lpdf', 'uniform_rng', 'variance',
+    'von_mises_lpdf', 'von_mises_rng', 'weibull_cdf', 'weibull_lccdf',
+    'weibull_lcdf', 'weibull_lpdf', 'weibull_rng', 'wiener_lpdf', 'wishart_lpdf',
+    'wishart_rng'
+  ]
+  const DISTRIBUTIONS = [
+    'bernoulli', 'bernoulli_logit', 'beta', 'beta_binomial', 'binomial',
+    'binomial_logit', 'categorical', 'categorical_logit', 'cauchy', 'chi_square',
+    'dirichlet', 'double_exponential', 'exp_mod_normal', 'exponential', 'frechet',
+    'gamma', 'gaussian_dlm_obs', 'gumbel', 'hypergeometric', 'inv_chi_square',
+    'inv_gamma', 'inv_wishart', 'lkj_corr', 'lkj_corr_cholesky', 'logistic',
+    'lognormal', 'multi_gp', 'multi_gp_cholesky', 'multi_normal',
+    'multi_normal_cholesky', 'multi_normal_prec', 'multi_student_t', 'multinomial',
+    'neg_binomial', 'neg_binomial_2', 'neg_binomial_2_log', 'normal',
+    'ordered_logistic', 'pareto', 'pareto_type_2', 'poisson', 'poisson_log',
+    'rayleigh', 'scaled_inv_chi_square', 'skew_normal', 'student_t', 'uniform',
+    'von_mises', 'weibull', 'wiener', 'wishart'
+  ]
+
   return {
+    aliases: ['stanfuncs'],
+    keywords: {
+      'title': BLOCKS.join(' '),
+      'keyword': STATEMENTS.concat(VAR_TYPES).concat(SPECIAL_FUNCTIONS).join(' '),
+      'built_in': FUNCTIONS.join(' ')
+    },
+    lexemes: hljs.IDENT_RE,
```

Comment:
Was UNDERSCORE_IDENT_RE incorrect?

Reply:
Ping.  If we figure this out I think we could get this merged into the next release. 

Did @jrnold disappear?  The auto-complete doesn't work. Strange.

Reply:
Oh, so in looking very closely our underscore is for identifiers that can START with underscore... boy it could use a better name IMHO. :-)